profiling-ffi/cbindgen.toml

```diff
@@ -41,6 +41,8 @@ renaming_overrides_prefixing = true
 "SendResult" = "ddog_prof_Exporter_SendResult"
 "SerializeResult" = "ddog_prof_Profile_SerializeResult"
 "Slice_File" = "ddog_prof_Exporter_Slice_File"
+"UpscalingRuleAddResult" = "ddog_prof_Profile_UpscalingRuleAddResult"
+"UpscalingInfo" = "ddog_prof_Profile_UpscalingInfo"
```

Comment:
Minor: This is no longer needed (since now the FFI exposes specific functions to add each upscaling variant)

Reply:
thx :) will clean this up